app/linode/main.go

```diff
@@ -67,7 +68,22 @@ func handle() {
 	mounter := mountmanager.NewSafeMounter()
 	deviceUtils := mountmanager.NewDeviceUtils()
 
-	ms, err := metadataservice.NewMetadataService(cloudProvider, *node)
+	// Get information about the Linode this pod is executing in.
+	// Assume that the Linode ID file does not exist.
+	linodeInfo := *node
```

Comment:
I think it would be cleaner to embed all of this logic inside `NewMetadataService`

Reply:
Agreed. I modified the code to match the original signature.